crates/orderbook/src/main.rs

```diff
@@ -273,6 +274,90 @@ struct Arguments {
     pub liquidity_order_owners: Vec<H160>,
 }
 
+impl std::fmt::Display for Arguments {
+    fn fmt(&self, f: &mut std::fmt::Formatter<'_>) -> std::fmt::Result {
+        write!(f, "{}", self.shared)?;
+        writeln!(f, "bind_address: {}", self.bind_address)?;
+        writeln!(f, "db_url: SECRET")?;
+        writeln!(f, "skip_event_sync: {}", self.skip_event_sync)?;
+        writeln!(
+            f,
+            "min_order_validity_period: {:?}",
+            self.min_order_validity_period
+        )?;
+        writeln!(
+            f,
+            "max_order_validity_period: {:?}",
+            self.max_order_validity_period
+        )?;
+        writeln!(f, "skip_trace_api: {}", self.skip_trace_api)?;
+        writeln!(
+            f,
+            "token_quality_cache_expiry: {:?}",
+            self.token_quality_cache_expiry
+        )?;
+        writeln!(f, "unsupported_tokens: {:?}", self.unsupported_tokens)?;
+        writeln!(f, "banned_users: {:?}", self.banned_users)?;
+        writeln!(f, "allowed_tokens: {:?}", self.allowed_tokens)?;
+        writeln!(f, "pool_cache_lru_size: {}", self.pool_cache_lru_size)?;
+        writeln!(f, "enable_presign_orders: {}", self.enable_presign_orders)?;
+        writeln!(
+            f,
+            "solvable_orders_max_update_age: {:?}",
+            self.solvable_orders_max_update_age
+        )?;
+        writeln!(f, "fee_discount: {}", self.fee_discount)?;
+        writeln!(f, "fee_factor: {}", self.fee_factor)?;
+        writeln!(
+            f,
+            "partner_additional_fee_factors: {:?}",
+            self.partner_additional_fee_factors
+        )?;
+        writeln!(f, "cow_fee_factors: {:?}", self.cow_fee_factors)?;
+        write!(f, "quasimodo_solver_url: ")?;
+        display_option(&self.quasimodo_solver_url, f)?;
```

Comment:
Doesn't the quasimodo URL also contain some sort of secret?

Reply:
Not a secret on kubernetes. Iirc it just points to a cluster internal url.